darts/models/forecasting/nbeats.py

```diff
@@ -760,7 +759,7 @@ def __init__(
             self.num_stacks = 2
 
         if isinstance(layer_widths, int):
-            self.layer_widths = [layer_widths] * num_stacks
+            self.layer_widths = [layer_widths] * self.num_stacks
```

Comment:
Could you also bring the same fix to `nhits.py` ?

Reply:
Done